### ISSUE TITLE
Added lr_schedulers.py

### ISSUE THREAD

**Issue by KaiyangZhou** [KaiyangZhou/deep-person-reid#112]
Learning rate scheduler is wrapped to `lr_schedulers.py` for easier management. Currently, it supports `single_step` and `multi_step`. For detailed changes, please refer to https://github.com/KaiyangZhou/deep-person-reid/commit/2a5a95166612e8bd459a672bf77352d1a572f343.
